konveyor-sample/src/main/java/com/example/konveyor/items/article/Article.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.konveyor.items.article
+
+import android.support.annotation.DrawableRes
+import android.support.annotation.StringRes
+import com.avito.konveyor.blueprint.Item
+
+data class Article(override val id: Long, val title: String, @StringRes val bodyPreview: Int, @DrawableRes val image: Int) : Item
```

Comment:
Why is it a data class? We don't use copy constructors or even equals/hashcode. Looks like unnecessary overhead.